src/Microsoft.AspNetCore.SignalR.Common/Internal/Encoders/Base64Encoder.cs

```diff
@@ -10,16 +10,17 @@ namespace Microsoft.AspNetCore.SignalR.Internal.Encoders
 {
     public class Base64Encoder : IDataEncoder
     {
-        public ReadOnlySpan<byte> Decode(byte[] payload)
+        public bool TryDecode(ref ReadOnlySpan<byte> buffer, out ReadOnlySpan<byte> data)
```

Comment:
Add a test, and also fix the compilation errors :P

Reply:
Yea doing this in VS code 😄. I'll spend more time looking at this soon.